sql/core/src/main/scala/org/apache/spark/sql/execution/DataSourceScanExec.scala

```diff
@@ -143,13 +143,15 @@ case class RowDataSourceScanExec(
 
     val topNOrLimitInfo =
```

Comment:
seems we don't need to change anything in `topNOrLimitInfo`

Reply:
I see.